sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Filtering/CollectionConfiguration.cs

```diff
@@ -118,16 +118,17 @@ private void UpdateAllErrorsWithKeyValue(CollectionConfigurationError[] errors,
 
         public string ETag => info.ETag;
 
-        private static void AddMetric<DocumentIngress>(
+        private static void AddMetric<TTelemetry>(
           DerivedMetricInfo metricInfo,
-          List<DerivedMetric<DocumentIngress>> metrics,
+          List<DerivedMetric<TTelemetry>> metrics,
           out CollectionConfigurationError[] errors)
+          where TTelemetry : DocumentIngress
```

Comment:
The rename here was for consistency to match the other files which use the name "TTelemetry".